integration-tests-scala/src/test/scala/tech/beshu/ror/integration/suites/base/BaseTemplatesSuite.scala

```diff
@@ -44,9 +51,9 @@ trait BaseTemplatesTests extends ForAllTestContainer with BeforeAndAfterEach {
 
   protected def templateExample(indexPattern: String): String = {
     val esVersion = rorContainer.esVersion
-    if(Version.greaterOrEqualThan(esVersion, 7, 0, 0)) {
+    if (Version.greaterOrEqualThan(esVersion, 7, 0, 0)) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Consider using case matching instead of else if blocks](https://app.codacy.com/manual/scarduzio/elasticsearch-readonlyrest-plugin/pullRequest?prid=5163287)